tests/test_helper.tcl

```diff
@@ -623,6 +625,8 @@ for {set j 0} {$j < [llength $argv]} {incr j} {
         incr j
     } elseif {$opt eq {--valgrind}} {
         set ::valgrind 1
+    } elseif {$opt eq {--sanitizer}} {
+        set ::sanitizer 1
```

Comment:
why not have that by default (when the build supports it)?

Reply:
What do you mean by "when the build supports it". If we enable it by default, it'll search sanitizer errors in the log even Redis is built without sanitizer instrumentation. I'm not sure if this is a problem though.(Just searchs stderr which is often empty I believe, so I expect it just adds slight performance cost). We can make it default.